content/online-resources/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Online Resources"
+weight: 4
+resources_section:
+  enable: true
+  title: "Project Websites"
+  content: "Find information about NWB on our various websites"
+  boxes:
+    - title: "Main Website"
+      content: "The primary NWB project website"
+      image: "/images/portfolio.png"
```

Comment:
I would suggest to use the NWB logo as the image to show show for the main website.